app/workers/enemy_worker.rb

```diff
@@ -1,67 +1,57 @@
-class EnemyWorker
+class EnemyWorker < ApplicationWorker
```

Comment:
Same here as other worker. Why do we need to derive from `ApplicationWorker` here?

Reply:
new Rails conventions...
- ApplicationController
- ApplicationRecord
- ApplicationMailer

you put all the common stuff for your workers in there...instead of duplicating it across all the workers.